docs/pages/blog/base-ui-2024-plans.md

```diff
@@ -36,7 +36,7 @@ Worry no more, as we're focused on shipping the first versions of all of the com
 Currently, Base UI can be customized to your heart's content using the `slots` and `slotProps` props.
 This API, while powerful, has proven to be less than ideal in some instances.
 Most notably, it's too lengthy and complicated to write and read when used with libraries such as Tailwind CSS.
-Additionally, since the `slots` and the corresponding `slotProps` are not related to Typescript types, it's possible to introduce bugs or have the compiler complain about valid code.
+Additionally, since the `slots` and the corresponding `slotProps` are not related to Typescript types, it's possible to introduce bugs or have the compiler complain about invalid valid code.
```

Comment:
```suggestion
Additionally, since the `slots` and the corresponding `slotProps` are not related to Typescript types, it's possible to introduce bugs or have the compiler complain about valid code.
```